vizro-core/examples/visual-vocabulary/pages/examples/gantt.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+
+tasks = pd.DataFrame(
+    [
+        {"Task": "Job A", "Start": "2009-01-01", "Finish": "2009-02-28"},
+        {"Task": "Job B", "Start": "2009-03-05", "Finish": "2009-04-15"},
+        {"Task": "Job C", "Start": "2009-02-20", "Finish": "2009-05-30"},
+    ]
+)
+
+
+page = vm.Page(
+    title="Gantt",
+    components=[vm.Graph(px.timeline(tasks.iloc[::-1], x_start="Start", x_end="Finish", y="Task"))],
```

Comment:
I would sort the dataframe by starting value. This makes it a bit more flexible for any data set as you normally visualise project timelines and need it in chronological order :) 

```suggestion
    components=[vm.Graph(figure=px.timeline(tasks.sort_values("Start", ascending=False), x_start="Start", x_end="Finish", y="Task"))],
```